files/www/cgi-bin/ports

```diff
@@ -368,7 +368,7 @@ if f then
 
             if _out:match("-") then
                 if validate_port_range(_out) then
-                    _in = _out:match("^(%d+)")
+                    _in = _out:match("^%s*(%d+)")
```

Comment:
I dont believe this is necessary here as any whitespace was removed earlier in the code (line 335)

Reply:
The `validate_port_range(_out)` at line 370 will remove white spaces inside the function but will only return true/false status. What is bellow is not aware of what has happen inside the function.

Then if true, `_in` will be set to the first digital part of `_out` but a white space at beginning will fail to properly set `_in` since we have the `^` indicating the beginning of the REGEX and it is expected to be one or more digit. If anything in front, it will fail to properly set the `_in` variable value. 

Test with white space in front of the port range (it can be noticed at the top values):

Without the %s*: 
![image](https://user-images.githubusercontent.com/13443249/208282053-65eb49a3-5d1f-4094-9f17-97abc4be6436.png)

With the %s*:
![image](https://user-images.githubusercontent.com/13443249/208282070-25a0a0af-349c-4ac9-ba14-a0d8d84cd365.png)

Reply:
I tested this fix on two of my test nodes and it worked fine.